--- src/app/components/Achievements/common.ts
+++ src/app/components/Achievements/common.ts
@@ -6,11 +6,23 @@
 import CritertecImg from '@/public/images/Critertec.svg';
+import SoyDigitalImg from '@/public/images/SoyDigital.svg';
 import { AchievementItem } from './types';
 
 export const achievements: AchievementItem[] = [
   {
-    name: 'Critertec - Soy Digital',
+    name: 'Critertec',
     key: 'critertec',
     description: 'achievements.description.critertec',
+    link: 'https://critertec.com/',
+    image: CritertecImg,
+    imageWidth: 300,
+    imageHeight: 200,
+    backgroundColor: '#ffffff',
+    isCustom: false,
+    col: 6,
+  },
+  {
+    name: 'Soy Digital RD',
+    key: 'soyDigital',
+    description: 'achievements.description.soyDigital',
     link: 'https://www.soydigital.gob.do/',
-    image: CritertecImg,
+    image: SoyDigitalImg,
     imageWidth: 300,
     imageHeight: 200,
     backgroundColor: '#ffffff',
     isCustom: false,
-    col: 12,
+    col: 6,
   },
